schemas/get-case.schema.json

```diff
@@ -368,19 +367,48 @@
       "description": "An explanation about the purpose of this instance.",
       "default": ""
     },
-    "nationality1": {
-      "$id": "#/properties/nationality1",
-      "type": "string",
-      "title": "The nationality1 schema",
-      "description": "An explanation about the purpose of this instance.",
-      "default": ""
-    },
-    "nationality2": {
-      "$id": "#/properties/nationality2",
-      "type": "string",
-      "title": "The nationality2 schema",
+    "phoneNumber": {
+      "$id": "#/properties/defendantPhoneNumber",
+      "type": [
+        "object",
+        "null"
+      ],
+      "title": "The phoneNumber schema",
       "description": "An explanation about the purpose of this instance.",
-      "default": ""
+      "default": {},
+      "required": [],
+      "properties": {
+        "mobile": {
+          "$id": "#/properties/defendantPhoneNumber/properties/mobile",
+          "type": [
+            "string",
+            "null"
+          ],
+          "title": "The mobile phoneNumber schema",
+          "description": "An explanation about the purpose of this instance.",
```

Comment:
Can we omit these descriptions if we have no more valuable information to add? Keeping this description boilerplate adds nothing so if it doesn't break anything we should get rid of it and keep the schema a bit cleaner.